packages/app-page-builder/src/editor/plugins/elementSettings/text/TextSettings.tsx

```diff
@@ -192,9 +193,9 @@ const TextSettings: React.FC<TextSettingsProps> = ({ defaultAccordionValue, opti
                         leftCellSpan={5}
                         rightCellSpan={7}
                     >
-                        <SelectField value={text.tag} onChange={updateTag}>
+                        <SelectField value={text.tag} onChange={updateTag} data-testid={"data-test-heading-type"}>
```

Comment:
To be more in-sync with the IDs below, let's also prefix this test ID with `text-settings` instead of `data-test`.

Reply:
Also, let's rename `heading-type` to `heading-select`.